conduct.md

```diff
@@ -100,32 +100,46 @@ TODO: Write an introductory paragraph.
 
 ### Conflicts
 
-TODO: Write an introductory paragraph.
+Students experiencing conflicts should come together to fix the problem. No argument
+should result in fighting, teams or groups should work together to solve any
+conflicts in the class. When dealing with conflicts if resolutions cannot be resolved
+then they should take conflicts to professor to resolve said issues. Lastly students
+should try to come together to peacefully resolve conflict.
 
 #### Acceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Students should try to understand the other persons point of view.
+- Students should talk directly and calmly to the other person.
+- Students should listen to everything that the other person is saying.
 
 #### Unacceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Students should not be rude when there is a conflict.
+- Students should not get angry at the other person.
+- Students should not raise their voice when there is a conflict.
 
 ### Infractions
 
-TODO: Write an introductory paragraph.
+Below, we will be outlining what is acceptable and unacceptable conduct. If followed,
+there should be no problems throughout the course of the year. Infraction is the act
+of breaking something, so for this class this will mean the act of
+breaking acceptable conduct. You should be able to follow all of the acceptable
+conducts in the class, and that will lead
```

Comment:
please add either 'you' or 'the students' after 'will lead' to show who this paragraph is referring to.